ocw/models.py

```diff
@@ -76,22 +77,17 @@ def all_time_fields(self):
             last_fmt = self.last_seen.strftime('%Y-%m-%d %H:%M')
         return all_time_pattern.format(self.age_formated(), first_fmt, last_fmt, self.ttl_formated())
 
-    def tags(self):
+    def set_alive(self):
+        self.last_seen = timezone.now()
+        self.active = True
+        self.age = self.last_seen - self.first_seen
+        if self.state != StateChoice.DELETING:
+            self.state = StateChoice.ACTIVE
         try:
-            info = json.loads(self.csp_info)
-            if 'tags' in info:
-                return info['tags']
-        except json.JSONDecodeError:
-            pass
-        return dict()
+            self.ignore = bool(self.cspinfo.tags[Instance.IGNORE_TAG_NAME])
```

Comment:
what about when `tags=None` ? I wanted to cover this case also and I assume in such case you will get ValueError

Reply:
> what about when `tags=None` ? I wanted to cover this case also and I assume in such case you will get ValueError

Do you mean `self.cspinfo.tags is None` ?  Because `bool(None) is False`.

Reply:
> > what about when `tags=None` ? I wanted to cover this case also and I assume in such case you will get ValueError
> 
> Do you mean when `self.cspinfo.tags` is `None` ? Because `bool(None)` is `False`.

yes but you can't call `None.get()` so you will get exception before `bool()` resolving

Reply:
ok thanks for clarification , than I will need to catch not ValueError but TypeError :) so I will switch to your suggested code but will keep try/catch still